includes/class-wc-validation.php

```diff
@@ -96,7 +96,9 @@ public static function is_postcode( $postcode, $country ) {
 			case 'NL':
 				$valid = (bool) preg_match( '/^([1-9][0-9]{3})(\s?)(?!SA|SD|SS)[A-Z]{2}$/i', $postcode );
 				break;
-
+			case 'SI':
+				$valid = (bool) preg_match( '/^([1-9][0-9]{3})$/i', $postcode );
```

Comment:
I believe the `i` modifier is not necessary as this expression matches only numbers?

Reply:
Hello, you are right. I overlooked this when modifying the NL case.